strings/group_anagram.py

```diff
@@ -5,22 +5,22 @@ def find_anagrams(_list: list):
     empty_dict = {}  # Python names variables using snake_case.
     empty_list = []
     for word in _list:
-        if "".join(sorted(word)) in emptyDict:
-            if emptyDict.get("".join(sorted(word))) != word:
-                emptyDict.update(
+        if "".join(sorted(word)) in empty_dict:
+            if empty_dict.get("".join(sorted(word))) != word:
+                empty_dict.update(
                     {
-                        "NA".join(sorted(word)): emptyDict.get("".join(sorted(word)))
+                        "".join(sorted(word)): empty_dict.get("".join(sorted(word)))
                         + ","
                         + word
                     }
                 )
         else:
-            emptyDict.update({"".join(sorted(word)): word})
-    for key, value in emptyDict.items():
+            empty_dict.update({"".join(sorted(word)): word})
+    for key, value in empty_dict.items():
         listword = value.split(",")
```

Comment:
```suggestion
        list_word = value.split(",")
```

Reply:
Hi @cclauss : pre-commit is getting failed, please help to fix

Reply:
Lines 5 and 12
```
strings/group_anagram.py:5: error: Need type annotation for "empty_dict" (hint: "empty_dict: Dict[<type>, <type>] = ...")
strings/group_anagram.py:12: error: Unsupported left operand type for + ("None")
strings/group_anagram.py:12: note: Left operand is of type "Optional[Any]"
Found 2 errors in 1 file (checked 1040 source files)
```